Doc update for slow operation detector [CTT-1152] (#2173)
hazelcast/hz-docs: docs/modules/ROOT/pages/system-properties.adoc

|int
|Defines a threshold above which a running operation in `OperationService` is considered to be slow.
These operations log a warning and are shown in the Management Center with detailed information, e.g., stacktrace.

> The value should be set at least 1000 ms for reliable results. The detector cannot detect operations under 1000 ms.

Reviewer: This is not quite accurate - it _can_ detect operations under this, it's just a gamble as to whether they are present at the time of captures being made (making results unreliable)

Reviewer 2: ```suggestion
The value should be set at least 1000 ms. The detector cannot reliably detect all operations taking under 1000 ms.
```
?

Author: Fixed at [78fe7f0](https://github.com/hazelcast/hz-docs/pull/2173/commits/78fe7f0d3bc0885a13d62caddb065e524456ea22)
